src/styles/_catalog.scss

```diff
@@ -1,10 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
 
   max-width: 944px;
   margin: 0 auto;
 
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 1fr);
```

Comment:
you can write it without media. it can be default(mobile first)

Reply:
default should be one column when width is less than 488